datasets/snli/README.md

```diff
@@ -0,0 +1,142 @@
+---
+language: 
+- en
+task:
+- text-classification
+purpose:
+- NLI
+size:
+- ">100k"
+language producers:
+- crowdsourced
+annotation:
+- crowdsourced
+tags:
+- extended-from-other-datasets
+license: "CC BY-SA 4.0"
```

Comment:
I love it, this will be very useful.

One thing about `size`: maybe we should use a range like "500k-1M"

Reply:
Do we need to support multiple values of `size` for a single dataset? Otherwise it should probably be 
```
size: ">100k"
```

(just a value, not an array)

Reply:
Some datasets have sub-datasets (glue, wikipedia, etc.) with different sizes so we can have several sizes

Reply:
does it make sense to have a list (not a map) of all sub-dataset sizes in that case though?

Reply:
I think so. e.g. KILT sub-tasks vary from a few thousand to a few million examples